docs/data/base/guides/interoperability-with-tailwind-css/interoperability-with-tailwind-css.md

```diff
@@ -0,0 +1,461 @@
+               <path d="M5 5v3.111c0 .491.398.889.889.889H9" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+             </svg>
+-          </button>
++          </Button>
+         </div>
+-        <button type="button" className="bg-white text-slate-900 dark:bg-slate-100 dark:text-slate-700 flex-none -my-2 mx-auto w-20 h-20 rounded-full ring-1 ring-slate-900/5 shadow-md flex items-center justify-center" aria-label="Pause">
++        <Button className="bg-white text-slate-900 dark:bg-slate-100 dark:text-slate-700 flex-none -my-2 mx-auto w-20 h-20 rounded-full border-2 border-slate-900/5 shadow-md flex items-center justify-center" aria-label="Pause">
+           <svg width="30" height="32" fill="currentColor">
+             <rect x="6" y="4" width="4" height="24" rx="2" />
+             <rect x="20" y="4" width="4" height="24" rx="2" />
+           </svg>
+-        </button>
++        </Button>
+         <div className="flex-auto flex items-center justify-evenly">
+-          <button type="button" aria-label="Skip 10 seconds">
++          <Button aria-label="Skip 10 seconds">
+             <svg width="24" height="24" fill="none">
+               <path d="M17.509 16.95c-2.862 2.733-7.501 2.733-10.363 0-2.861-2.734-2.861-7.166 0-9.9 2.862-2.733 7.501-2.733 10.363 0 .38.365.711.759.991 1.176" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+               <path d="M19 5v3.111c0 .491-.398.889-.889.889H15" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+             </svg>
+-          </button>
+-          <button type="button" className="hidden sm:block lg:hidden xl:block" aria-label="Next">
++          </Button>
++          <Button className="hidden sm:block lg:hidden xl:block" aria-label="Next">
+             <svg width="24" height="24" fill="none">
+               <path d="M14 12 6 6v12l8-6Z" fill="currentColor" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+               <path d="M18 6v12" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+             </svg>
+-          </button>
+-          <button type="button" className="rounded-lg text-xs leading-6 font-semibold px-2 ring-2 ring-inset ring-slate-500 text-slate-500 dark:text-slate-100 dark:ring-0 dark:bg-slate-500">
++          </Button>
++          <Button className="rounded-lg text-xs leading-6 font-semibold px-2 border-2 border-slate-500 text-slate-500 dark:text-slate-100 dark:ring-0 dark:bg-slate-500 hover:ring-cyan-500">
+             1x
+-          </button>
++          </Button>
+         </div>
+       </div>
+     </div>
+```
+
+Some classes were slightly changed on some buttons so we have a consistent focus indicator.
+
+## Conclusion
```

Comment:
This is a silly pet peeve of mine 😛 but I don't like using the word "Conclusion" as the header for the conclusion of an article.

```suggestion
## What we learned
```